sfja/src/components/AddStudent/Input.js

```diff
@@ -17,12 +17,13 @@ const textWidth = {
 
 // eslint-disable-next-line require-jsdoc
 function blankStateExceptSubmitTime(submitTime) {
+  const today = new Date();
   return {
     submitTime: submitTime,
     firstNameStudent: '',
     middleNameStudent: '',
     lastNameStudent: '',
-    dob: new Date(),
+    dob: today.toLocaleDateString(),
```

Comment:
Maybe I should consolidate this to `new Date().toLocaleDateString()`

Reply:
@jchin01 thoughts?

Reply:
I think you should consolidate to new Date().toLocaleDateString()

Reply:
I was trying to figure out how to suggest it in code but im not too sure how

Reply:
All boxes in Github are in Markdown
https://guides.github.com/features/mastering-markdown/

To write a single line of code, surround the code with a tick (`).
To write multiple lines of code, surround the code with three ticks, newline, code, then three more ticks.

Reply:
I'll put a new commit up!

Reply:
Fixed with 2c730fa! Resolving.